src/qibocal/protocols/ramsey/ramsey_signal.py

```diff
@@ -306,6 +306,9 @@ def _update(
 ):
     if results.detuning is not None:
         update.drive_frequency(results.frequency[target][0], platform, target)
+        platform.calibration.single_qubits[
+            target
+        ].qubit.frequency_01 = results.frequency[target]
```

Comment:
```suggestion
        ].qubit.frequency_01 = results.frequency[target][0]
```
Since as you see above the frequency is a tuple containing the value and the error.

Reply:
true. Corrected.